queen-attack/example.nim

```diff
@@ -1,18 +1,12 @@
-type
-  Position* = tuple[rank, file: int]
-
-proc isValid(white, black: Position): bool =
+proc isValid(white, black: tuple[rank, file: int]): bool =
```

Comment:
I guess it depends how these exercises are done, I must admit I'm not entirely sure. Indeed, it's not a big deal. Feel free to keep it as it is :)